model/DataStore/DeliveryMetadataListener.php

```diff
@@ -49,12 +49,20 @@ public function whenDeliveryIsPublished(Event $event): void
         try {
             $this->logDebug(sprintf('Processing MetaData event for %s', get_class($event)));
             $this->checkEventType($event);
-            $params['deliveryId'] = $event->getDeliveryUri();
-            $params[self::OPTION_MAX_TRIES] = $this->getOption(self::OPTION_MAX_TRIES, 10);
-            $this->triggerSyncTask($params);
+            $this->triggerSyncTask([
+                MetaDataDeliverySyncTask::DELIVERY_OR_TEST_ID_PARAM_NAME => $event->getDeliveryUri(),
+                MetaDataDeliverySyncTask::INCLUDE_DELIVERY_METADATA_PARAM_NAME => true,
+                MetaDataDeliverySyncTask::MAX_TRIES_PARAM_NAME => $this->getOption('max_tries', 10),
```

Comment:
I think `OPTION_MAX_TRIES` constant would still be useful, because it is used here

Reply:
Fixed